_1327/main/forms.py

```diff
@@ -55,30 +57,62 @@ def clean(self):
 
 
 class MenuItemCreationForm(MenuItemForm):
+	group = forms.ModelChoiceField(Group.objects.all(), label=_('Edit permissions'), disabled=False, required=True)
```

Comment:
we should think about putting all of this group handling stuff in the forms into a common place where it can also be used by the document forms